truffle.js

```diff
@@ -46,7 +79,8 @@ module.exports = {
     simulated_live: {
```

Comment:
can you make additional network called `nano` and leave this intact?

Reply:
on that